### PR TITLE
use new wholesale gem

### LINKED ISSUE #2: Wholesale Users
Users should be flagged as Wholesale and see wholesale prices.

### PR DESCRIPTION
**Pull request by thogg4**
closes #2

### CODE REVIEW

**Comment by thogg4** [comment]
@ericsaupe can you check this out?

File: Vagrantfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ Vagrant.configure('2') do |config|
   end
 
   config.vm.synced_folder "../spree-blogging-spree", "/spree-blogging-spree", :nfs => true
+  config.vm.synced_folder "../spree_wholesale", "/spree_wholesale", :nfs => true
```
Comment (ericsaupe) [inline Vagrantfile]: Do these need to be commented out by default, or does it matter?

**Comment by ericsaupe** [comment]
Nice work, @thogg4. I knew that gem was going to need to be forked for 3.0. Looks like you got to it first. I updated your initial comment to be `Fixes #2`. With that wording when we merge this pull request Github will automatically close that issue as well. https://github.com/blog/1506-closing-issues-via-pull-requests

:panda_face:

File: Vagrantfile (hunk 1 above)
Comment (thogg4) [inline Vagrantfile]: It doesn't matter

**Comment by thogg4** [comment]
@ericsaupe this one is a whole new gem, so maybe you should check that out as well.
Also I think just referencing #2 will close the issue.

**Comment by thogg4** [comment]
Actually it looks like you need a keyword. https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by ericsaupe** [comment]
@thogg4 looks good. I actually think its better you rewrote it. The other one I saw was totally unmaintaned.

**Pull request merged**